packages/global/components/site-header/contextual.marko

```diff
@@ -7,8 +7,8 @@ $ const { config, site, req } = out.global;
 $ const blockName = input.blockName || "site-header";
 
 $ const contexts = site.getAsArray("navigation.contexts");
-$ const contextualNav = contexts.find(({ when }) => asArray(when).some(path => isActiveLink(req.path, path)));
-
+$ const pageType = input.pageType
```

Comment:
This should also use object destructuring...

`$ const { pageType } = input`

Reply:
@zarathustra323  Should this be merged with line 7 into

`$ const { blockName = "site-header", pageType } = input;`